iwyu.cc

```diff
@@ -3997,6 +3999,22 @@ int main(int argc, char **argv) {
   std::unique_ptr<clang::CompilerInstance> compiler(CreateCompilerInstance(
       options_parser.clang_argc(), options_parser.clang_argv()));
   if (compiler) {
+    // Figure out the absolute path to the resource-dir, which might be relative
+    // to the install directory
+    StringRef Executable = argv[0];
+    llvm::SmallString<128> AbsoluteResourceDir(IWYU_RESOURCE_DIR);
```

Comment:
The name is a little weird, as this is most likely relative at this point?

Reply:
It's not necessarily, it's whatever is set in the cmake settings. At this point, an absolute setting would most likely indicate building locally.